Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -806,9 +959,20 @@ public void CreateRig()
 
         #endregion
 
+
         #region MonoBehaviour Methods
+
         private void Start()
         {
+            //override Shell Parity settings if no prefab is specified for corner handles;
+            if (scaleHandlePrefab == null)
+            {
+                closeScale = 1.8f;
```

Comment:
the defaults for these fields are different than the values here - I would prefer we stick with one default setup

Reply:
These are optimized handle size values for the default blue (HL1 style) handles when no handle prefabs are assigned. When custom handle prefabs including HL2 shell-parity handles are assigned, BB uses the default or assigned values through the inspector or code. With single default values, we couldn't achieve a proper scale.

Reply:
Instead of magic numbers could we move these to fields or constants in the file?